packages/dapp-tic-tac-toe/src/Wager.jsx

```diff
@@ -1,22 +1,193 @@
-import React, { Component } from 'react';
-import { ReactComponent as Logo } from './assets/images/logo.svg';
-import { Link } from "react-router-dom";
+import React, { Component } from "react";
+import { ReactComponent as Logo } from "./assets/images/logo.svg";
 
 class Wager extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      user: {},
+      opponent: {},
+      intermediary: null
+    };
+  }
+
+  componentDidMount() {
+    this.props.cfProvider.once("install", this.onInstall.bind(this));
+
+    // TODO: This token should be obtained from LocalStorage.
+    const token =
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjQ5OGRhZTNmLWNmYjctNGNmNC05OTZiLWZiNDI5NDI3ZGQ4NSIsInVzZXJuYW1lIjoiam9lbCIsImVtYWlsIjoiZXN0dWRpb0Bqb2VsYWxlamFuZHJvLmNvbSIsImFkZHJlc3MiOiIweDBmNjkzY2M5NTZkZjU5ZGVjMjRiYjFjNjA1YWM5NGNhZGNlNjAxNGQiLCJtdWx0aXNpZ0FkZHJlc3MiOiIweDE0NTczMjUzMTkxRDJDMjUxQTg1Y0JBMTQ1NjY0RWUwYUViNDA4NjgiLCJpYXQiOjE1NDcwODU4MTcsImV4cCI6MTU3ODY0MzQxN30.AQ-ataiWl9emPRWtHVinEXYgyHHZquP9DOXLjmcTKJI";
+
+    fetch(
+      "https://server.playground-staging.counterfactual.com/api/matchmake",
```

Comment:
TODO: This URL should come from an environment variable and get injected with something similar to what the Playground does with `rollup-plugin-replace`.